packages/commonwealth/client/scripts/controllers/chain/ethereum/contracts/adapter.ts

```diff
@@ -0,0 +1,66 @@
+import { EthereumCoin } from 'adapters/chain/ethereum/types';
+
+import EthereumAccount from 'controllers/chain/ethereum/account';
+import EthereumAccounts from 'controllers/chain/ethereum/accounts';
+import { ChainBase } from 'common-common/src/types';
+import { IChainAdapter, ChainEntity, ChainEvent, NodeInfo, ChainInfo } from 'models';
+
+import ChainEntityController from 'controllers/server/chain_entities';
+import { IApp } from 'state';
+
+import { notifyError } from 'controllers/app/notifications';
+import { AaveTypes } from 'chain-events/src';
+import AaveChain from '../aave/chain';
+
+export default class Aave extends IChainAdapter<EthereumCoin, EthereumAccount> {
```

Comment:
What is this? Can we delete this file?

Reply:
Yes can delete this, I'll delete!

Reply:
this was a POC of a design that we eventually rejected